cilium/socket_option_source_address.h

```diff
@@ -84,9 +84,14 @@ class SourceAddressSocketOption : public Network::Socket::Option,
 
     // Note: we don't expect this to change the behaviour of the socket. We expect it to be copied
     // into the upstream connection later.
```

Comment:
Is this comment still valid?

Reply:
still valid - but the whole `setOption` is executed on the upstream socket.

I slightly adjusted the comment.